www/lynx/Portfile

```
@@ -18,6 +18,8 @@ long_description    Lynx is a fully-featured World Wide Web browser for \
                     emulators.
 homepage            https://lynx.invisible-island.net
 
+depends_build       port:perl5
```

The perl5 port presumably isn't required; macOS perl has been sufficient until now. To prevent perl5 from being installed unnecessarily when macOS perl exists, you could rewrite this as:
```suggestion
depends_build       bin:perl:perl5
```

---

Right, so the port fails to install under trace mode without adding a depends_build on perl:

```
sed -n -e '/Config_Type  *Config_Table/,/{0, *0, *0}/ p' ./src/LYReadCFG.i | \
        sed  -e 's/ *{ *"\([^"]*\)".*/\1/' | \
        perl ./scripts/cfg2html.pl -ams ./lynx.cfg
perl: posix_spawn: /opt/local/var/macports/sip-workaround/0/usr/bin/perl5.28: No such file or directory
make: *** [alphatoc.html] Error 1
make: Leaving directory `/opt/local/var/macports/build/_Users_rak_macports-ports_www_lynx/lynx/work/lynx2.8.9rel.1'
Command failed:  cd "/opt/local/var/macports/build/_Users_rak_macports-ports_www_lynx/lynx/work/lynx2.8.9rel.1" && /usr/bin/make -w install install-doc install-help DESTDIR=/opt/local/var/macports/build/_Users_rak_macports-ports_www_lynx/lynx/work/destroot
Exit code: 2
Warning: The following existing files were hidden from the build system by trace mode:
  /opt/local/bin/perl
  /private/var/select/sh
  /var/root/.CFUserTextEncoding
The following files would have been hidden from the build system by trace mode if they existed:
  /Network/Library/Preferences/com.apple.versioner.perl.plist
  /usr/gnu/include
  /usr/local/cuda
  /usr/local/cuda-7.0
  /usr/local/cuda-7.5
  /usr/local/cuda-8.0
  /var/root/Library/Preferences/com.apple.versioner.perl.plist
Error: Failed to destroot lynx: command execution failed
```

The change you proposed works. I've force pushed it.